src/debug/HyprCtl.cpp

```diff
@@ -742,6 +742,10 @@ std::string rollinglogRequest(eHyprCtlOutputFormat format, std::string request)
 std::string globalShortcutsRequest(eHyprCtlOutputFormat format, std::string request) {
     std::string ret       = "";
     const auto  SHORTCUTS = g_pProtocolManager->m_pGlobalShortcutsProtocolManager->getAllShortcuts();
+
+    if (SHORTCUTS.empty())
+        return "no global shortcuts exist";
```

Comment:
phrasing could be better as this might be misleading. `"No apps have registered any global shortcuts"`